ReadMe.md

```diff
@@ -166,6 +169,7 @@ Front-end | フロントエンド | |
 Management | 管理 | かんり | Of people, software, etc
 Manager | 管理者 | かんりしゃ |
 Infrastructure | インフラ | |
+System administrator | インフラ |
```

Comment:
Oh never heard that use before. I see why you structured it the way you did. let me double check with another source first before I get back to you on this one